### PR TITLE
Added static offline reference

### PR DESCRIPTION
**Pull request by mkatychev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ Built with [Zola](https://www.getzola.org/), the static site generator written i
 zola serve
 ```
 
+Build static offline copy:
+
+```sh
+zola build --base-url="${PWD}/public/index.html" --output-dir=./public
```
Comment (mkatychev) [inline README.md]:
```suggestion
zola build --base-url=./public/index.html --output-dir=./public 
```

**Comment by ralfbiedert** [comment]
Hi, thanks. Unfortunately this command doesn't work on Windows. Also, I'd like to keep the `README` relatively clean. 

How about adding a `FAQ.md` instead, with a section "How can I build an offline copy?", with a sentence of the explanation of the command, and then something like "For example on Linux: ..."?

**Comment by mkatychev** [comment]
> Hi, thanks. Unfortunately this command doesn't work on Windows. Also, I'd like to keep the `README` relatively clean.
> 
> How about adding a `FAQ.md` instead, with a section "How can I build an offline copy?", with a sentence of the explanation of the command, and then something like "For example on Linux: ..."?

done

**Comment by ralfbiedert** [comment]
Thanks!

**Pull request merged**